packages/atoms/globals.css

```diff
@@ -3,8 +3,21 @@
  * When using this component in any Cal project, the globals are automatically imported
  * in that project.
  */
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+ @tailwind base;
+ @tailwind components;
+ @tailwind utilities;
+ 
+ @import "../ui/styles/shared-globals.css";
 
-@import "../ui/styles/shared-globals.css";
+ ::-webkit-scrollbar{
+    width: 10px; 
+  }
+  
+  ::-webkit-scrollbar-track{
+    background-color: aliceblue;
+  }
```

Comment:
This property is not compatible with all browsers. (Firefox, Safari for mobile)

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar